src/app/notes/notes.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Note } from "./note.model";
+import { EventEmitter } from '@angular/core';
+
+export class NotesService {
+    noteSelected = new EventEmitter<Note>();
```

Comment:
eventEmitter bedzie Ci potrzebny do komunikacji miedzy komponentami w serwisie jest on raczej niepotrzebny

Reply:
mozesz zamiast tego dodac metoda addNote(note) ktora dorzuci to do pola Notes

Reply:
znaczy nie bardzo rozumiem, bo tu miałem zamysł żeby klikniętą notke ustawić jako noteSelected, ale to nie będzie dodawać nic do notes tylko będzie emitować tego klika i wyświetlać noteSelected obok listy. Więc nie chce dodawać nic do tablicy (do dodawania notek będzie oddzielny komponent), tylko chce ją wyemitować po evencie clicka

Reply:
więc w tej sytuacji nadal powinienem to robic metodą czy EventEmitter jest ok ?